applications/StructuralMechanicsApplication/custom_elements/timoshenko_beam_element_2D3N.h

```diff
@@ -0,0 +1,363 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: StructuralMechanicsApplication/license.txt
+//
+//  Main authors:  Alejandro Cornejo
+//
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_elements/timoshenko_beam_element_2D2N.h"
+#include "custom_utilities/structural_mechanics_element_utilities.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class LinearTimoshenkoBeamElement2D3N
+ * @ingroup StructuralMechanicsApplication
+ * @brief This is the Timoshenko beam element of 3 nodes. 5th order degree Hermitic polynomials
+ * for deflection and 2nd degree for axial displacements. The ordering of the local shape functions
+ * assume a " 0--1--2 " ordering and then we swap the components when moving to global coordinates
+ *
+ *                                    ^ y, v
+ *                                    |
+ * Global Ordering of the nodes:      0 ------ 2 ------- 1 --> x, u      and rotation theta node-wise
+ * Reference: Felippa and Oñate,
+ * "Accurate Timoshenko Beam Elements For Linear Elastostatics and LPB Stability",
+ * Archives of Comp. Methods in Eng. (2021) 28:2021-2080
+ * DOI: https://doi.org/10.1007/s11831-020-09515-0; Then adapted to the 3 noded straight beam element.
+ * @author Alejandro Cornejo
+ */
+class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) LinearTimoshenkoBeamElement2D3N
+    : public LinearTimoshenkoBeamElement2D2N
+{
+
+public:
+
+    ///@name Type Definitions
+    ///@{
+
+    /// The base element type
+    using BaseType = LinearTimoshenkoBeamElement2D2N;
+
+    // Counted pointer of BaseSolidElement
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(LinearTimoshenkoBeamElement2D3N);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    // Constructor void
+    LinearTimoshenkoBeamElement2D3N()
+    {
+    }
+
+    // Constructor using an array of nodes
+    LinearTimoshenkoBeamElement2D3N(IndexType NewId, GeometryType::Pointer pGeometry) : BaseType(NewId, pGeometry)
+    {
+        mThisIntegrationMethod = GeometryData::IntegrationMethod::GI_GAUSS_5;
+    }
+
+    // Constructor using an array of nodes with properties
+    LinearTimoshenkoBeamElement2D3N(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties)
+        : BaseType(NewId,pGeometry,pProperties)
+    {
+        // This is needed to prevent uninitialised integration method in inactive elements
+        mThisIntegrationMethod = GeometryData::IntegrationMethod::GI_GAUSS_5;
```

Comment:
better to initialize the var directly where it is defined

this way e.g. when a new ctor is added it wont be forgotten